includes/class-token-user.php

```diff
@@ -245,4 +245,21 @@ public function find_token_users( $refresh = false ) {
 		}
 		return $user_ids;
 	}
+
+	/**
+	 * Find tokens by UUID.
+	 *
+	 * @param string $uuid UUID.
+	 * @return array Results.
+	 */
+	public function find_by_uuid( $uuid, $user = null ) {
```

Comment:
Could we use a more meaningful name? (Here and elsewhere)

```suggestion
	public function find_by_consent_id( $uuid, $user = null ) {
```